### DISCUSSION TITLE
Perhaps a version for ChromeOS?

### DISCUSSION THREAD

**Discussion by piklok** [ntsc-rs/ntsc-rs#213] [Ideas]
Considering the existence of a version for other Linux instances, perhaps a ChromeOS version could be developed as well?

**Comment by piklok** [comment]
Another thing: A means of manually adjusting the amount of dot crawl in renders?

**Comment by fuki98** [comment]
WTF?

**Comment by TheSystemGuy1337** [comment]
**All Chromebooks are ewaste and are worse than eMachines computers.**

**Comment by Degamisu** [comment]
In response,

I don't think a ChromeOS version of NTSC-RS could become a thing, as not many people use it, and it would be a burden to compile. Virtual machines may help unless your Chromebook is a `school issued computer`, in that case it is up to your school's policy which may not be allowed.

A dot crawl adjuster would technically not be possible in the real world. The demodulators and filters can help minimize, but not remove these analog artifacts. In all I do not think this would be added

Regards,
Degamisu.

**Comment by valadaptive** [comment]
If you own your Chromebook (and not your school), you can [install Linux on it](https://support.google.com/chromebook/answer/9145439?hl=en) and run ntsc-rs that way.

ChromeOS itself can only run glorified webpages, so creating ntsc-rs for ChromeOS would amount to, essentially, creating a web version of ntsc-rs. The ability to transform video files on the web [does exist](https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API), but is very immature and not well-supported. Maybe in a couple years, if support is better, I'll reconsider. Right now, it doesn't seem technically feasible.

**Reply by TheSystemGuy1337** [reply]
But here's the issue: Linux under ChromeOS will be slower as it has to be virtualized, which then hampers the performance of NTSC-RS. And most Chromebooks (in my personal experience) are slow and likely don't have proper virtualization support, making it even slower. And you can't run Linux natively on them (wipe ChromeOS out and replace it with Linux) because all Chromebooks have a proprietary bootstrap ROM which cannot be flashed with a standard UEFI/IBM PC compatible BIOS on most models and so far no discoveries have been made to jailbreak the bootstrap ROM to enable booting of other OSes besides ChromeOS. *most* Chromebooks even then tend to be on the lower end of hardware specs; NTSC-RS would be so slow it would end up being unusable. I see no benifit to porting NTSC-RS to ChromeOS or even offering it as a web app, even if that was possible.

For context on just how slow a Chromebook can be, I tried running QEMU to run Windows XP on a Chromebook I owned before. It didn't even get to the second stage of the setup before softlocking. And it can't even run Windows 98. All other systems I owned could easily run XP under a VM.


I'm sorry if my previous response was considered "disruptive" but that was just my personal opinion.
